### PR TITLE
MAISTRA-880 updated formatting in smcp options docs

### PR DESCRIPTION
**Pull request by dmitri-d**

### CODE REVIEW

File: topics/docs/installation/installation-options.adoc [hunk 1]
```diff
@@ -178,8 +219,12 @@ These are the resources requested and may vary depending on your environment. Th
 [options="header"]
 |=======
 |Parameter |Description |Default
-|cpu|This is the number of CPUs that are requested in the environment.  | 1000m
-|memory| This is the ammount of memory that is requested in the environment. | 1G
+|`cpu`
+|This is the number of CPUs that are requested in the environment.  
+| 1000m
+|`memory`
+|This is the ammount of memory that is requested in the environment.
```
Comment (brian-avery) [inline topics/docs/installation/installation-options.adoc]: amount

**Review by brian-avery — APPROVED** [review]

File: topics/docs/installation/installation-options.adoc (hunk 1 above)
Comment (dmitri-d) [reply topics/docs/installation/installation-options.adoc]: Fixed here and everywhere else.

File: topics/docs/installation/installation-options.adoc [hunk 2]
```diff
@@ -20,7 +20,9 @@ charts. Many of the parameters supported by the installer are shown below.
 [options="header"]
 |=======
 |Parameter |Description |Default
-|template|A template references a ServiceMeshControlPlane referenced in the smcp-templates ConfigMap from which configuration should be inherited. | default
+|`template`
+|A template references a `ServiceMeshControlPlane` referenced in the `smcp-templates` `ConfigMap` from which configuration should be inherited. 
+|default
```
Comment (jwendell) [inline topics/docs/installation/installation-options.adoc:25]: Worth to add a link to docs/installation/controlplane-templates/ ?

File: topics/docs/installation/installation-options.adoc [hunk 3]
```diff
@@ -55,25 +57,39 @@ charts. Many of the parameters supported by the installer are shown below.
...
+`disablePolicyChecks` must be false for 3scale to work.
+{{% /notice %}} 
+| true
+|`policyCheckFailOpen` 
+|This is a boolean indicating wehther to traffic should be allowed through when the mixer policy service cannot be reached.  
+| false
+|`hub`
+|The hub to use to pull the Istio images.
+|`maistra/` if deployment_type is `origin` and `openshift-service-mesh/` if deployment_type is `openshift`
+|`tag`
```
Comment (jwendell) [inline topics/docs/installation/installation-options.adoc:67]: can we remove this legacy ansible stuff?

File: topics/docs/installation/installation-options.adoc [hunk 4]
```diff
@@ -55,25 +57,39 @@ charts. Many of the parameters supported by the installer are shown below.
...
 |Parameter |Description |Default
-|image | This parameter controls the image to use for proxy-init.
+|`image` 
+|This parameter controls the image to use for `proxy-init`.
 {{% notice info %}}
-This must be changed to proxy-init-centos7 if running on a RHEL 7 host.
-{{% /notice %}} 1.0.0 | proxy-init-ubi8
+This must be changed to `proxy-init-centos7` if running on a `RHEL 7` host.
+{{% /notice %}} 1.0.0 
+| proxy-init-ubi8
```
Comment (jwendell) [inline topics/docs/installation/installation-options.adoc]: This is not used at all on OCP4, correct? I mean, this parameter is totally ignored as OCP4 uses CNI.
Should we remove this from this table?

or @brian-avery can remove this as part of his pr that removes 3.11 stuff...

File: topics/docs/installation/installation-options.adoc (hunk 3 above)
Comment (dmitri-d) [reply topics/docs/installation/installation-options.adoc:67]: removed.

File: topics/docs/installation/installation-options.adoc (hunk 4 above)
Comment (dmitri-d) [reply topics/docs/installation/installation-options.adoc]: removed.

File: topics/docs/installation/installation-options.adoc (hunk 2 above)
Comment (dmitri-d) [reply topics/docs/installation/installation-options.adoc:25]: Added a link to docs.

File: topics/docs/installation/installation-options.adoc [hunk 5]
```diff
@@ -20,7 +20,9 @@ charts. Many of the parameters supported by the installer are shown below.
 [options="header"]
 |=======
 |Parameter |Description |Default
-|template|A template references a ServiceMeshControlPlane referenced in the smcp-templates ConfigMap from which configuration should be inherited. | default
+|`template`
+|A template references a `ServiceMeshControlPlane` referenced in the `smcp-templates` `ConfigMap` from which configuration should be inherited. See link:{{< ref "controlplane-templates.adoc" >}}[contemplate documentation] for details. 
+|default
```
Comment (jwendell) [inline topics/docs/installation/installation-options.adoc:25]: contemplate typo

File: topics/docs/installation/installation-options.adoc [hunk 6]
```diff
@@ -55,34 +57,36 @@ charts. Many of the parameters supported by the installer are shown below.
...
-
-### proxy_init
-
-[options="header"]
-|=======
-|Parameter |Description |Default
-|image | This parameter controls the image to use for proxy-init.
-{{% notice info %}}
-This must be changed to proxy-init-centos7 if running on a RHEL 7 host.
-{{% /notice %}} 1.0.0 | proxy-init-ubi8
```
Comment (jwendell) [inline topics/docs/installation/installation-options.adoc:76]: The example still has references to `proxy_init`.

Comment (dmitri-d) [reply topics/docs/installation/installation-options.adoc:76]: removed.

File: topics/docs/installation/installation-options.adoc (hunk 5 above)
Comment (dmitri-d) [reply topics/docs/installation/installation-options.adoc:25]: fixed

**Review by jwendell — APPROVED** [review]

**Pull request merged**